driver/src/main/scala/com/stratio/sparkta/driver/factory/CuratorFactoryHolder.scala

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.driver.factory
+
+import akka.event.slf4j.SLF4JLogging
+import com.datastax.driver.core.policies.DefaultRetryPolicy
+import com.typesafe.config.{Config, ConfigException}
+import org.apache.curator.RetryPolicy
+import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory, CuratorTempFramework}
+import org.apache.curator.retry.{RetryNTimes, ExponentialBackoffRetry}
+
+import scala.Option
+
+/**
+ * Customized factory that encapsulates the real CuratorFrameworkFactory and creates a singleton instance of it.
+ * @author anistal
+ */
+object CuratorFactoryHolder extends SLF4JLogging {
+
+  private var curatorFramework: Option[CuratorFramework] = None
+
+  /**
+   * Default ZK connection parameters when no config was specified.
+   */
+  val DefaultConnectionString   =   "localhost:2181"
```

Comment:
I think it should be in config.properties.template or something like this. What do you think?